apis/infix-en_GB/extensions/kotlin_1_3/atrium-api-infix-en_GB-kotlin_1_3-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/kotlin_1_3/ResultFeatureAssertionsSpec.kt

```diff
@@ -0,0 +1,60 @@
+package ch.tutteli.atrium.api.infix.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import ch.tutteli.atrium.specs.integration.ResultFeatureAssertionsSpec
+
+class ResultFeatureAssertionsSpec : ResultFeatureAssertionsSpec(
+    ("toBe ${success::class::simpleName}" to (Companion::toBeSuccessFeature)).withFeatureSuffix(),
+    "toBe ${success::class::simpleName}" to Companion::toBeSuccess,
+    ("toBe ${success::class::simpleName}" to (Companion::toBeSuccessFeatureNullable)).withNullableSuffix(),
```

Comment:
add withFeatureSuffix  in addition
```suggestion
    ("toBe ${success::class::simpleName}" to (Companion::toBeSuccessFeatureNullable)).withFeatureSuffix().withNullableSuffix(),
```